build_tools/azure/posix.yml

```diff
@@ -37,6 +37,7 @@ jobs:
     TEST_DOCSTRINGS: 'false'
     CREATE_ISSUE_ON_TRACKER: 'false'
     SHOW_SHORT_SUMMARY: 'false'
+    SELECTED_TESTS: $[ dependencies.git_commit.outputs['commit.selected'] ]
```

Comment:
Rather than having `git_commit` parses the commit message and pass it here, can we have a new step in `poxis.yml` that parses the commit message and pass it along to the next steps?

The upside is that the test configuration is responsible for parsing the commit to select tests and not the `git_commit` job. The downside is that we need to repeat a little bit of code for extracting the commit, but I think it's easier for future maintainers compared to using a `dependencies.git_commit`.